tests/helpers.py

```diff
@@ -27,6 +27,26 @@
 from engine.tolerance import tolerance
 
 
+def setup_test_directory(directory_path):
```

Comment:
See comment above

Reply:
I would still keep it, right?

Reply:
For what exactly do you need the path `tests/tmp`?
Does a test operate in it?

When refactoring the tests I paid attention, that all tests do not modify anything in the repo itself.
All tests write intermediate files or result into the folder created by the `tmp_path` fixture only.

Couldn't you stick to that policy for this tests as well?
This would save us another fixture.

Reply:
Yes, makes sense to change it to `tmp_path`. The policy makes sense, I was just not aware of it.